lib/lte_link_control/lte_lc_helpers.h

```diff
@@ -141,14 +141,19 @@
 #define AT_CONEVAL_DL_PATHLOSS_INDEX		17
 
 /* MDMEV command parameters */
-#define AT_MDMEV_ENABLE				"AT%%MDMEV=1"
+#define AT_MDMEV_ENABLE				"AT%%MDMEV=2"
```

Comment:
@trantanen @tokangas Any preference on how to handle this, where mfw <= 1.3.x doesn't support type 2? Should we just take the simplest route and try `2` first, if that fails, try `1` in all cases?

Reply:
@jtguggedal I think it's fine to always try `2` first and then fall back to `1` in case the first attempt fails.

Reply:
Alright, I went for that